### DISCUSSION TITLE
Can I add a select input on header or sidebar?

### DISCUSSION THREAD

**Discussion by felipechiodini** [filamentphp/filament#6048] [Help]
Hello, i am building an admin panel and I wish dispaly an select to change the store that user is managing at moment, does have any solution? I already search on documentation.

![image](https://user-images.githubusercontent.com/50850744/228403825-e253a0ea-5e44-49fd-8df7-592de751230b.png)

**Comment by danharrin** [comment]
Yes, search "render hooks" in the docs
